src/txHashBuilder.c

```diff
@@ -342,18 +347,35 @@ void txHashBuilder_addOutput_tokenGroup(
 void txHashBuilder_addOutput_token(
         tx_hash_builder_t* builder,
         const uint8_t* assetNameBuffer, size_t assetNameSize,
-        uint64_t amount
+        uint64_t amount,
+        bool includeScriptDataHash
```

Comment:
Don't we store this in `tx_hash_builder_t`?

Reply:
Indeed we do.